pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -0,0 +1,109 @@
+"""Plugin for constructor definitions."""
+import astroid
+from pylint.checkers import BaseChecker
+from pylint.interfaces import IAstroidChecker
+from pylint.lint import PyLinter
+
+
+class HassConstructorFormatChecker(BaseChecker):  # type: ignore[misc]
+    """Checker for setup definitions."""
+
+    __implements__ = IAstroidChecker
+
+    name = "hass_enforce_type_hints"
+    priority = -1
+    msgs = {
+        "W0012": (
+            "First argument should be of type HomeAssistant",
+            "hass-enforce-type-hints-hass",
```

Comment:
I'd remove `-enforce-` part to make these names a bit shorter.

Reply:
Done